TESTING.md

```diff
@@ -148,68 +148,84 @@ Running tests against an iOS device locally is not currently supported.
 
 ### React-native
 
-The React-native tests drive real, remote mobile devices using BrowserStack. As a Bugsnag employee you can access the necessary credentials in our shared password manager.
+The React-native tests drive real, remote mobile devices using BrowserStack. As a Bugsnag employee you can access the 
+necessary credentials in our shared password manager.
 
-In addition, the react-native test fixture requires the current version of the repo to be published to a locally owned NPM repository (! Not the official NPMJS repository !). This can be locally or remotely hosted, but should be versioned appropriately.  Three bits of information will need to be passed into the test run as environment variables in order to access this package:
+The test fixture (a React Native app) that tests are run against needs to be built with a published version of 
+@bugsnag/react-native.  By default, the build process will base this on the current branch/comment, 
+e.g. `7.2.0-my-branch.231f6ef7`.  This can be overridden using the environment variable NOTIFIER_VERSION, which is useful during development when 
+making test, but not notifier, changes.
 
+If building against the current branch/commit, the packages must be published to a locally owned NPM repository 
```

Comment:
It would be helpful to have a copy/pasteable command that publishes to a local verdaccio or something here. It was unclear as to whether the `build-react-native-*` commands would do that for me, until I ran them and they complained about the version not being available.

Reply:
I used your script `node ./scripts/publish.js http://localhost:4873`

Reply:
Note added - including a reference to the Verdaccio website.